src/test/java/de/tum/in/test/integration/ReflectionTestUtilsTest.java

```diff
@@ -24,18 +24,22 @@ class ReflectionTestUtilsTest {
 	private final String testGetMethod_success = "testGetMethod_success";
 	private final String testInvokeMethod_invocationTarget = "testInvokeMethod_invocationTarget";
 	private final String testInvokeMethod_success = "testInvokeMethod_success";
```

Comment:
Nice to see more tests added 👍 One thing to note here that - I admit - is not currently documented: The tests are executed by method name, and to have everything consistent, all fields and test methods are ordered by name, too. Could you instruct your IDE to sort fields and methods here and in `ReflectionTestUtilsUser` as well? This *should* only reorder the new ones from this PR.

Reply:
Done, it looks like some existing methods changed order in `ReflectionTestUtilsUser`, too.